code/game/objects/items/miscellaneous.dm

```diff
@@ -144,6 +144,56 @@
 			augment_list[initial(A.name)] = A
 	return augment_list
 
+/obj/item/choice_beacon/syndicate_trooper
+	name = "Syndicate Trooper Equipment Box"
+	desc = "Contains your equipment."
+
+/obj/item/choice_beacon/syndicate_trooper/generate_display_names()
+	var/static/list/equipment_list
+	if(!equipment_list)
+		equipment_list = list()
+		var/list/templist = list(
+		/obj/item/storage/box/syndicate_trooper/hardliners,
+		/obj/item/storage/box/syndicate_trooper/republic)
+		for(var/V in templist)
+			var/atom/A = V
+			equipment_list[initial(A.name)] = A
+	return equipment_list
+
+/obj/item/storage/box/syndicate_trooper
+	name = "Syndicate Trooper Equipment Box"
+	desc = "Contains your equipment."
+
+/obj/item/storage/box/syndicate_trooper/hardliners
+	name = "Gorlex Hardliners Trooper Equipment Box"
+
+/obj/item/storage/box/syndicate_trooper/hardliners/PopulateContents()
+	new /obj/item/clothing/under/syndicate/hardliners(src)
+	new /obj/item/clothing/suit/armor/hardliners(src)
+	new /obj/item/clothing/head/helmet/hardliners(src)
+	new /obj/item/radio/headset/syndicate/alt(src)
+	new /obj/item/storage/belt/security/webbing/hardliners(src)
+	new /obj/item/clothing/glasses/hud/security/sunglasses/hardliners(src)
+	new /obj/item/clothing/mask/balaclava/combat(src)
+	new /obj/item/clothing/gloves/combat(src)
+	new /obj/item/clothing/shoes/combat(src)
+	new /obj/item/storage/box/survival/security(src)
+
+/obj/item/storage/box/syndicate_trooper/republic
+	name = "New Gorlex Republic Trooper Equipment Box"
+
+/obj/item/storage/box/syndicate_trooper/republic/PopulateContents()
+	new /obj/item/clothing/under/syndicate/ngr(src)
+	new /obj/item/clothing/suit/armor/ngr(src)
+	new /obj/item/clothing/head/helmet/ngr(src)
+	new /obj/item/radio/headset/syndicate/alt(src)
+	new /obj/item/storage/belt/security/webbing/ngr(src)
+	new /obj/item/clothing/glasses/hud/security/sunglasses/ngr(src)
+	new /obj/item/clothing/mask/balaclava/ngr(src)
+	new /obj/item/clothing/gloves/combat(src)
+	new /obj/item/clothing/shoes/combat(src)
+	new /obj/item/storage/box/survival/security(src)
+
```

Comment:
it'd be nice if a few more factions were available since we're making this loosey goosey anyways

off the top of my head; SUNS peacekeeper and inteq enforcer (maybe)

Reply:
when i said i was cool with having troopers add tension i did not mean that they necessarily cause tension

the trooper is just any grunt that was available from one of the factions; there is no reason why SUNS, at the least, should not have an available slot

i kinda wish we had ACLF/Cybersun trooper roles

i'll roll over on inteq though thats probably silly